docs/manuals/run-deploy/deploy/azure/index.md

```diff
@@ -1,108 +1,7 @@
-[Azure](https://azure.microsoft.com/en-us), one of the world's leading providers of cloud services,
-is a very relevant choice for deploying Taipy applications. It allows developers to stay focused on
-their application bringing effortless scalability, monitoring and security natively. The following
-documentation shows how to deploy on Azure App Service and get an application into the hands of the
-end-user.
+Taipy applications are designed to be easily deployable and customizable. Once deployed, they can natively support multiple users with authentication and authorization. Taipy is cloud-agnostic and can be deployed on any cloud provider or on-premises.
 
-## Prerequisites
+!!! note "Taipy Enterprise edition"
 
-- Minimal knowledge of Azure.
-- Azure CLI should be installed. Check
-  [the official documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for
-  Azure CLI installation.
-- [:material-arrow-right: Running a Taipy application](../../run/index.md)
+    This part of the documentation is available on demand. Please contact us using the link below. Taipy Deployment Support, Scripts and Documentation are readily available in the Enterprise edition.
```

Comment:
This is not what we said.
1. The doc is available on demand, and not only for Enterprise Edition customers.
2. The Enterprise Edition customers benefit from 2 days free of charge of consulting and support to deploy user applications with them.

Reply:
That's what was discussed yesterday.

Reply:
Let me ask Des' input on this before going forward with this

Reply:
Vincent agrees with the idea of not precising what the number of days of support we provide in the Enterprise license. I'll go ahead and create a new PR for the other changes if that's okay with you.

Reply:
Yep
My proposal was this: 
    Besides this documentation page, Enterprise Edition customers benefit from 
    free-of-charge consulting and support to deploy user applications. Check out the
    [Taipy Enterprise Edition](http://toto.tata.com).